packages/zkdb/src/merkle-tree/merkle-tree-stream.ts

```diff
@@ -10,22 +10,26 @@ export type TMerkleTreeData = {
 export class MerkleTreeReadStream extends Readable {
   private merkleTreeData: TMerkleTreeData;
   private currentIdx: number = 0;
+  private batchSize: number;
 
-  constructor(data: TMerkleTreeData) {
+  constructor(data: TMerkleTreeData, batchSize = 10) {
     super({ objectMode: true });
     this.merkleTreeData = data;
+    this.batchSize = batchSize;
   }
 
   _read() {
```

Comment:
```suggestion
  private _read() {
```

Reply:
I am getting an error which tells that the classes extended incorrectly when adding `private`